src/changes-backwards-compatible.apib

```diff
@@ -6,6 +6,8 @@ We list all backwards-compatible additions here. These are currently available i
 
 ### Latest
 
+- As events can be linked to deals, we added the `deal` link type to `events.list` and `events.info`, and allow filtering on deals in `events.list`.
```

Comment:
The new `deal` link type and filter was added in #507 and hence is mentioned in the changes for February 2020. We should only adjust that change's description and indeed add the `deal` link type for the info endpoint.

So, I think we should rather reword the change in February instead of listing this as a new change that is yet to be released (which is not true).